app/controllers/api_controller.rb

```diff
@@ -17,7 +17,7 @@ def index
   private
 
   def entrypoint_versions
-    version_config[:definitions].select(&:ident).collect do |version_specification|
+    Api::Settings.version[:definitions].select(&:ident).collect do |version_specification|
```

Comment:
They can all use `Api::Settings` - it's unfortunate that this controller is the only one that needs to since it can't be under the `Api` namespace without overriding some defaults in the routes. Maybe that would be worth doing, though?